src/ui/units/datasets/store/constants.ts

```diff
@@ -47,9 +47,12 @@ export const initialPreview: DatasetReduxState['preview'] = {
 };
 
 export const initialState: DatasetReduxState = {
+    isUpdatingDataset: false,
```

Comment:
Actually it is fetching (according to the place set this option in)

Btw, it called `isRefetchingDataset` directly at the place of use